application/src/main/java/org/togetherjava/tjbot/features/utils/RequestFailedException.java

```diff
@@ -0,0 +1,19 @@
+package org.togetherjava.tjbot.features.utils;
+
+public class RequestFailedException extends Exception {
```

Comment:
i dont quite see the point of introducing a checked exception and an unchecked variant at the same time. just go with the unchecked

Reply:
I need to use an unchecked exception in JShellAPI because it doesn't support checked exceptions (more specifically ResponseUtils.ofJson), the unchecked exception is purely internal and not leaked outside of JShellAPI.
I use a checked exception when using JShellAPI but they must be handled, and I already forgotten several times during the coding of the commands to handle them, so it's too important to make it unchecked.

Reply:
mh, fair